src/ClientSample/Program.cs

```diff
@@ -18,9 +18,18 @@ namespace Microsoft.FactoryOrchestrator.ClientSample
     /// 
     /// </summary>
     public static class FactoryOrchestratorNETCoreClientSample
-    {
+    {
+        /// <summary>
+        /// Gets the failed TaskRun guids. This is used for Factory Orchestrator testing only, not by this sample code.

```

Comment:
a little confusing, perhaps explain how its used?

Reply:
I don't want to go into testing details in the sample code as it might create even more confusion (I want the sample to focus on being a sample), but I added a pointer to the src/Tests/ClientSampleIntegrationTest/ClientSampleIntegrationTests.cs file and added more context there if the sample user wants more info.